src/Opulence/Framework/Databases/Console/Commands/RunDownMigrationsCommand.php

```diff
@@ -55,21 +56,68 @@ protected function define()
      */
     protected function doExecute(IResponse $response)
     {
-        if ($this->optionIsSet('number')) {
-            $numMigrations = (int)$this->getOptionValue('number');
+        try {
+            if ($this->optionIsSet('number')) {
+                $numMigrations = (int)$this->getOptionValue('number');
 
-            if ($numMigrations === 1) {
-                $response->writeln('Rolling back last migration...');
+                $migrationsRolledBack = $this->doExecuteSome($response, $numMigrations);
             } else {
-                $response->writeln("Rolling back last $numMigrations migrations...");
+                $migrationsRolledBack = $this->doExecuteAll($response);
             }
 
-            $migrationsRolledBack = $this->migrator->rollBackMigrations($numMigrations);
+            $this->writeResults($response, $migrationsRolledBack);
+        } catch (Exception $e) {
+            $this->writeException($response, $e);
```

Comment:
I don't think this would ever be hit in the current implementation because we're swallowing any exceptions thrown during migration.  However, if we go with my suggestion above of letting the exceptions bubble, this works.